Cargo.lock

```diff
@@ -3734,6 +3734,7 @@ dependencies = [
  "percent-encoding",
  "pin-project-lite",
  "rustls",
+ "rustls-native-certs",
```

Comment:
I wonder why this addition, it shouldn't be needed. I suppose cargo will drop it later on.

Reply:
Tried a cargo check of the branch locally, and it gets removed. Not an issue though!